CefSharp.Core/Internals/RenderClientAdapter.h

```diff
@@ -167,7 +167,11 @@ namespace CefSharp
                     bitmapInfo->NumberOfBytes = numberOfBytes;
                 }               
 
-                CopyMemory(backBufferHandle, (void*)buffer, bitmapInfo->NumberOfBytes);
+                if (!bitmapInfo->DirtyRectSupport || ((bitmapInfo->DirtyRect.Width != 0) && (bitmapInfo->DirtyRect.Height != 0)))
```

Comment:
I'm a little surprised that `CEF` is calling `OnPaint` with `Width == 0` and `Height == 0`. Is it called very often? Probably should check the `CEF` source, change should really be applied there, no point calling `OnPaint` if there's nothing to draw.